tools/roadmap/validate_dag.py

```
@@ -0,0 +1,68 @@
+    return nodes, edges_hard, edges_soft
+
+def gh_json(args):
+    out = run(["gh"] + args)
+    return json.loads(out)
+
+def main():
+    if not MMD.exists():
+        print("validate_dag: missing docs/ROADMAP-DAG.mmd", file=sys.stderr)
+        return 1
+    text = MMD.read_text(encoding='utf-8')
+    nodes, edges_hard, _ = parse_mmd(text)
```

Ruff (0.14.3)

Unpacked variable `edges_hard` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)